.github/workflows/build.yml

```diff
@@ -0,0 +1,62 @@
+name: Build SDK for latest API version
+
+on: 
+  pull_request:
+    branches:
+      - test-workflow
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+    
+    steps:
+    - uses: actions/checkout@master
+      with:
+        ref: 'refactored-build-plus-gemfile'
+      
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6'
+
+    - name: Install bundler
+      run: gem install bundler
+      
+    - name: Install dependencies
+      run: bundle install
+        
+    - name: Build new SDK version
+      run: ./Scripts/build 2020-04
+    
+    - name: Install xcpretty
+      run: gem install xcpretty      
+    
+    - name: Test generated schema
+      run: |
+        CURRENT_SIMULATOR_UUID=$(xcrun simctl create TestDevice com.apple.CoreSimulator.SimDeviceType.iPhone-11 com.apple.CoreSimulator.SimRuntime.iOS-13-3)
+        export CURRENT_SIMULATOR_UUID
+        ./Scripts/test_buy
+        ./Scripts/test_buy_tv
+        ./Scripts/test_buy_watch
+        ./Scripts/test_pay
+       
+    - name: Create and reset release branch
+      run: git checkout -B test-2020-04 # -B will reset branch if it exists
+      
+    - name: Stage and push generated files
+      run: |
+        git add ./Buy/Generated
+        git commit -m "Test 2020-04"
+        git push --set-upstream origin test-2020-04
+    
+##    - uses: actions/github-script
```

Comment:
Do we still need this? If not remove

Reply:
Yes, I'm still working on the tests. This is just commented out because it's improper syntax since it's not done.